src/quote.js

```diff
@@ -44,6 +44,10 @@ class Quote {
 
   get expiresAt () { return this._expiresAt; }
 
+  get timeOfRequest () { return this._timeOfRequest || new Date().getTime(); }
```

Comment:
This doesn't make sense. `quote.timeOfRequest` would just return the current time all the time.

I would set `this._timeOfRequest = new Date().getTime()` in the quote constructor. That will be overridden in SFOX subclass.